notes/macros.sty

```diff
@@ -303,6 +318,5 @@
 
 \newcommand{\domain}{\Omega}
 \newcommand{\epi}{\mathrm{epi}}
-\DeclareMathOperator*{\argmin}{\arg\!\min}
```

Comment:
Please do not edit macros.sty. This is breaking other files and preventing them from compiling. This macro for instance is used in lecture2.tex and elsewhere.